spec/job_spec.rb

```diff
@@ -37,6 +37,21 @@
     end
   end
 
+  context '#job_plays' do
+    let(:url)            { "example.com/api/v1/job_plays" }
+    let(:job_collection) { build(:response_collection, :klass => described_class) }
+    let(:job_events)     { build(:response_url_collection, :klass => AnsibleTowerClient::JobEvent, :url => url) }
+    it "returns a collection of AnsibleTowerClient::JobEvents" do
```

Comment:
@bzwei - Yes - it should be - but interestingly enough - I'm getting job_events back - so I'm digging into why.  

I was thinking the endpoint /jobs/5/job_plays/2 would actually return a JobEvent - it isn't - the related field points to job_events.  Trying to see why our Gem is returning a different type.